partiql-parser/src/lexer.rs

```diff
@@ -592,8 +600,8 @@ pub enum Token<'input> {
     Using,
     #[regex("(?i:Value)")]
     Value,
-    #[regex("(?i:Values)")]
-    Values,
```

Comment:
Looks like `Values` was accidentally removed

Reply:
Good catch, not sure how it's happened, will restore.

Reply:
Looks like the related commit hasn't been picked up.